src/main/java/com/minecolonies/rankup/Rankup.java

```diff
@@ -180,71 +186,38 @@ public void addReloadable(String module, Action reloadable)
      *
      * @throws IOException if the config could not be read.
      */
+    @SuppressWarnings("squid:S3655")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S3655' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)